airbyte-integrations/connectors/destination-dynamodb/src/test/java/io/airbyte/integrations/destination/dynamodb/DynamodbDestinationStrictEncryptTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.dynamodb;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import org.junit.jupiter.api.Test;
+
+public class DynamodbDestinationStrictEncryptTest {
+
+  private static final String NON_SECURE_URL_ERR_MSG = "Server Endpoint requires HTTPS";
+
+  /**
+   * Test that checks if user is using a connection that is HTTPS only
+   */
+  @Test
+  public void checksCustomEndpointIsHttpsOnly() {
```

Comment:
This is good, can we also just have a sanity check test for `https` variants return an `AirbyteConnectionStatus.SUCCEEDED`, this captures the scenario where maybe the custom endpoint gets augmented and we fail on `https` in the future

Reply:
Added one more test. Thanks